joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 10
 
effort_controller_snake:
  type: "effort_controllers/JointEffortController"
  joint: snake_joint
 
effort_controller_scorpion:
  type: "effort_controllers/JointEffortController"
  joint: scorpion_joint